java/java.lsp.server/nbcode/nbproject/platform.properties

```diff
@@ -30,7 +30,6 @@ cluster.path=\
 disabled.modules=\
     bcpg,\
     bcprov,\
-    com.google.guava,\
```

Comment:
Who needs Guava? I don't think NetBeans code is using it directly...

Reply:
It was added by mistake, it is not needed, removing